libs/libs-pomfix/test/test.js

```diff
@@ -0,0 +1,57 @@
+var test = require('tap').test
+var join = require('path').join
+
+var util = require('../lib/util')
+var fix = require('../lib/fix')
+
+var options = {
+  include: {
+    group: [/^org\.codice/, /^d../]
+  }
+}
+
+var run = function (prj) {
+  var dir = join(__dirname, 'resources', prj)
+  return util.find(dir)
+    .pipe(util.read())
+    .pipe(fix(util.load, options))
+}
+
+test('catch feature errors', function (t) {
+  t.plan(3)
```

Comment:
It's just the conventional name for the test obj the testing framework passes the function.